src/constants/constants.ts

```diff
@@ -2,3 +2,5 @@
 export const SESSION_COOKIE_EXPIRATION_DURATION_MS = 7 * 24 * 60 * 60 * 1000; // 7 days
 
 export const IFRAME_RESIZE_HEIGHT_COOKIE_EXPIRATION_DAYS = 365; // 365 days
+
+export const DEFAULT_LINK_SHOW_BUTTON = true;
```

Comment:
Isn't it overkill to have it as a constant (instead of e.g. a default prop)? Where is this used?

Reply:
Is is used by both builder and player, and as a default prop on the LinkItem in ui. I thought that putting it as a constant would let set it in one place and it would be synched across all places it is used ?